### PR TITLE
[YUNIKORN-113] [webapp] Change default value for maximum queue capacity

### PR DESCRIPTION
**Pull request by kingamarton**

### CODE REVIEW

**Comment by yangwwei** [comment]
hi @akhilpb001 could you help to review this change?

File: src/app/services/scheduler/scheduler.service.ts [hunk 1]
```diff
@@ -96,7 +96,7 @@ export class SchedulerService {
                     alloc['allocationKey'],
                     alloc['allocationTags'],
                     alloc['uuid'],
-                    this.formatCapacity(this.splitCapacity(alloc['resource'])),
+                    this.formatCapacity(this.splitCapacity(alloc['resource'], 'N/A')),
```
Comment (akhilpb001) [inline src/app/services/scheduler/scheduler.service.ts:99]: Change 'N/A' to 'n/a'

File: src/app/services/scheduler/scheduler.service.ts [hunk 2]
```diff
@@ -179,34 +179,34 @@ export class SchedulerService {
     const maxCap = data['capacities']['maxcapacity'] as string;
     const absUsedCapacity = data['capacities']['absusedcapacity'] as string;
 
-    const configCapResources = this.splitCapacity(configCap);
-    const usedCapResources = this.splitCapacity(usedCap);
-    const maxCapResources = this.splitCapacity(maxCap);
+    const configCapResources = this.splitCapacity(configCap, 'N/A');
```
Comment (akhilpb001) [inline src/app/services/scheduler/scheduler.service.ts:182]: Change 'N/A' to 'n/a'

File: src/app/services/scheduler/scheduler.service.ts [hunk 3]
```diff
@@ -217,7 +217,11 @@ export class SchedulerService {
 
   private formatCapacity(resourceInfo: ResourceInfo) {
     const formatted = [];
-    formatted.push(`[memory: ${CommonUtil.formatMemory(+resourceInfo.memory)}`);
+    if (resourceInfo.memory !== 'N/A') {
```
Comment (akhilpb001) [inline src/app/services/scheduler/scheduler.service.ts:220]: Change 'N/A' to 'n/a'

**Pull request closed without merging**